bundler/spec/dependabot/bundler/helper_spec.rb

```diff
@@ -14,7 +14,7 @@
     LOCKFILE
   end
 
-  let(:lockfile_bundled_with_v1) do
+  let(:first_version_lockfile) do
```

Comment:
v1 refers to the version of Bundler here. Not it being the first occurrence of the variable.

Reply:
I can change that one to lockfile_bundle_with_version_one.  It keeps the same spirit of that original name.  In another fie I changed v6 to version_six for that reason.

Reply:
corrected with commit 4c83ce7.  Please review